controllers/topics.controller.js

```diff
@@ -23,4 +23,14 @@ exports.getEndpoints = (req, res, next) => {
     res.status(200).send({endpoints});
 }
 
+exports.getComments = (req, res, next) => {
+    const article_id = req.params.article_id;
+    fetchComments(article_id).then(({rows}) => {
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.